Bottom: 7a9841082b213f84bacfb9b98f02942612eadf86
Top:    b32e7bc1ac2111ad1bac924b1876488fa8206f72
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 13:52:58 +0000

Refresh of remove-debug-from-hairline

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 8f6f681..b652377 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1660,10 +1660,7 @@ jump (VNODE **curv, DIRECTION *cdir, J_Rule j_rule)
   if (!(*curv)->cvc_prev)	/* not a cross-vertex */
     {
       if (VERTEX_DIRECTION_EDGE (*curv, *cdir)->Flags.mark)
-        {
-          g_warning ("Nowhere to go at normal-vertex");
-	  return FALSE;
-        }
+	return FALSE;
       return TRUE;
     }
 #ifdef DEBUG_JUMP
@@ -1696,7 +1693,6 @@ jump (VNODE **curv, DIRECTION *cdir, J_Rule j_rule)
 	}
     }
   while ((d = d->prev) != start); /* Keep searching around the cvc vertex for a suitable exit edge */
-  g_warning ("Nowhere to go at cross-vertex");
   return FALSE;
 }
